packages/framework/src/Model/Product/Search/Export/ProductSearchExportWithFilterRepository.php

```diff
@@ -65,7 +65,7 @@ public function getProductsData(int $domainId, string $locale, int $startFrom, i
                 'ean' => $product->getEan(),
                 'name' => $product->getName($locale),
                 'description' => $product->getDescription($domainId),
-                'shortDescription' => $product->getShortDescription($domainId),
+                'short_description' => $product->getShortDescription($domainId),
```

Comment:
This is a bugfix? I would appriciate if the change is separated into a standalone commit that describes reason for the change (together with changes in tests, upgrade instructions)

Reply:
This is kind of an bug fix that I need to fix in order to compare existing structure with the one in *.json. Do you really think that its necessary to move it to separate commit?

Reply:
if it is a bugfix, then definitely yes. The commit will be atomic and fulfill our https://github.com/shopsys/shopsys/blob/master/docs/contributing/guidelines-for-creating-commits.md#what-makes-a-good-commit

Reply:
done